server/models/db.js

```diff
@@ -0,0 +1,84 @@
+import dotenv from 'dotenv';
```

Comment:
You need to move all the queries to another file, And let this file be for only connecting to the database